xml/obs/openSUSE:Slowroll:Staging-images.xml

```diff
@@ -0,0 +1,10 @@
+<openQA
+    project_pattern="openSUSE:Slowroll:Staging"
+    dist_path="images"
+    distri="opensuse"
+    archs="x86_64">
+    <flavor name="JeOS-for-kvm-and-xen" folder="*/kiwi-templates-Minimal:kvm-and-xen">
+        <!-- Due to there being multiple archs, it omits "head -n1". So don't match .sha256(.asc). -->
+        <hdd filemask=".*kvm-and-xen.*\.qcow2$"/>
+    </flavor>
```

Comment:
```suggestion
    <flavor name="Slowroll" distri="opensuse" iso="1" folder="*product*"/>
```
I would expect something like that. Why flavor is `JeOS-for-kvm-and-xen`?

Reply:
Basically the flavor can be selected arbitrary just that according openQA scenario definitions or job templates need to exist. For example for Leap 15.5 Images https://openqa.opensuse.org/admin/job_templates/103 shows sections with key "products" like "opensuse-Leap15.5-JeOS-for-kvm-and-xen-x86_64"